library/src/main/java/me/grantland/widget/AutofitHelper.java

```diff
@@ -232,8 +262,9 @@ else if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN) {
     private float mMaxTextSize;
     private float mPrecision;
 
-    private boolean mEnabled;
+    private boolean mIsAutofitWidthEnabled;
     private boolean mIsAutofitting;
+    private static boolean mIsAutofitHeightEnabled;
```

Comment:
I believe the field is `static` so that it can be accessed via the `private static void autofit(TextView view, TextPaint paint, float minTextSize, float maxTextSize, int maxLines, float precision)` method. I'm not sure that the method needs to be `static` though. @grantland?